### PR TITLE
onepassword_raw - Add missing parameter to doc string

### LINKED ISSUE #5506: onepassword_raw; V2 (general 6.0.0); domain was not defined in configuration.
### Summary

Hi.

I have just upgraded to General 6.0.0 which includes support for 1Password V2.

The same Ansible configuration from v1 throws:
"An unhandled exception occurred while running the lookup plugin 'community.general.onepassword_raw'. Error was a <class 'KeyError'>, original message: 'Requested entry (plugin_type: lookup plugin: ansible_collections.community.general.plugins.lookup.onepassword_raw setting: domain ) was not defined in configuration.'. 'Requested entry (plugin_type: lookup plugin: ansible_collections.community.general.plugins.lookup.onepassword_raw setting: domain ) was not defined in configuration.'"

### Issue Type

Bug Report

### Component Name

community/general/plugins/lookup/onepassword_raw.py

### Ansible Version

```console (paste below)
$ ansible --version

```
ansible [core 2.13.5]
...
  python version = 3.8.10 (default, Jun 22 2022, 20:18:18) [GCC 9.4.0]
  jinja version = 3.1.2
  libyaml = True

### Community.general Version

```console (paste below)
$ ansible-galaxy collection list community.general

```
community.general 6.0.0

### Configuration

```console (paste below)
$ ansible-config dump --only-changed

```
- hosts: localhost
  vars:
    record: "my_record"
    vault: "my_vault"
  tasks:
    - set_fact:
        onepassword_items: "{{ lookup('community.general.onepassword_raw', record, vault=vault) | default({}, true) }}"
    - debug:
        msg: "{{ onepassword_items }}"

### OS / Environment

Distributor ID: Ubuntu
Description:    Ubuntu 20.04.5 LTS
Release:        20.04
Codename:       focal

### Steps to Reproduce

```yaml (paste below)

```
eval $(op signin)
ansible-playbook my_playbook.yml

### Expected Results

I expect the contents of the record in the vault to display.

### Actual Results

```console (paste below)

```
"An unhandled exception occurred while running the lookup plugin 'community.general.onepassword_raw'. Error was a <class 'KeyError'>, original message: 'Requested entry (plugin_type: lookup plugin: ansible_collections.community.general.plugins.lookup.onepassword_raw setting: domain ) was not defined in configuration.'. 'Requested entry (plugin_type: lookup plugin: ansible_collections.community.general.plugins.lookup.onepassword_raw setting: domain ) was not defined in configuration.'"

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by samdoran**
##### SUMMARY

Fixes #5506 

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

`plugins/lookup/onepassword_raw.py`

##### ADDITIONAL INFORMATION
This might fail sanity tests since the version_added is 6.0.0 which is already release. Though I'm curious how this passed sanity tests originally...

### CODE REVIEW

File: changelogs/fragments/5506-onepassword_raw-missing-param.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+   - onepassword_raw - Add missing parameter to plugin documentation (https://github.com/ansible-collections/community.general/issues/5506)
```
Comment (felixfontein) [inline changelogs/fragments/5506-onepassword_raw-missing-param.yml:2]:
```suggestion
   - onepassword_raw - add missing parameter to plugin documentation (https://github.com/ansible-collections/community.general/issues/5506).
```

File: plugins/lookup/onepassword_raw.py [hunk 2]
```diff
@@ -30,6 +30,11 @@
         description: Item section containing the field to retrieve (case-insensitive). If absent will return first match from any section.
       subdomain:
         description: The 1Password subdomain to authenticate against.
+      domain:
+        description: Domain of 1Password. Default is U(1password.com).
```
Comment (felixfontein) [inline plugins/lookup/onepassword_raw.py:34]: It's already shown next to the option.
```suggestion
        description: Domain of 1Password.
```

**Comment by felixfontein** [comment]
> This might fail sanity tests since the version_added is 6.0.0 which is already release. Though I'm curious how this passed sanity tests originally...

No worry, https://github.com/ansible/ansible/pull/69291 never got finished, so it won't fail :)

**Comment by samdoran** [comment]
Thanks for all your help @felixfontein!

File: changelogs/fragments/5506-onepassword_raw-missing-param.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+   - onepassword_raw - add missing parameter to plugin documentation (https://github.com/ansible-collections/community.general/issues/5506)
```
Comment (felixfontein) [inline changelogs/fragments/5506-onepassword_raw-missing-param.yml:2]:
```suggestion
   - onepassword_raw - add missing parameter to plugin documentation (https://github.com/ansible-collections/community.general/issues/5506).
```

**Comment by felixfontein** [comment]
@samdoran thanks for this one as well!

**Pull request merged**